e2e-tests/gef/cypress/e2e/clone/clone-gef-deal.spec.js

```diff
@@ -139,7 +140,7 @@ context('Clone GEF (AIN) deal', () => {
 
       cy.login(BANK1_MAKER1);
 
-      cy.get(`[data-cy="deal__link--${AINdealId}"]`).click();
+      cy.get(`[data-cy="deal__link--index--1 deal__link--${AINdealId}"]`).click();
```

Comment:
Can we use a selector from `dashboardDeals`?

Reply:
`dashboardDeals` from `Portal`?

Reply:
We could add it to GEF's `dashboard-page` selectors

Reply:
Added selector to GEF `dashboard-page`